src/plugins/CaptionsPlugin.js

```diff
@@ -59,17 +76,13 @@ export class CaptionsPlugin extends ButtonPlugin {
       }.bind(this)
     );
 
-    if (!this.captionsButton) {
-      console.warn(
-        'SpringRollContainer: CaptionPlugin was not provided a button element'
-      );
+    if (null === this.captionsButton) {
       return;
     }
 
-    this.captionsButton.addEventListener(
-      'click',
-      this.captionsButtonClick.bind(this)
-    );
+    this.captionsButton.classList.remove('disabled');
+    this.captionsMuted = !!SavedData.read(CAPTIONS_MUTED);
```

Comment:
This is still tying `captionsButton` to `captionsMuted` and `setCaptionsStyles` again. Can we invert the `if` statement on line 79?